lib/index.js

```diff
@@ -104,7 +104,7 @@ PostHTML.prototype.process = function (tree, options) {
   // sync mode
   if (options.sync === true) {
     this.plugins.forEach(function (plugin) {
-      api(tree)
+      Api.call(tree)
```

Comment:
`tree = new API(tree)` ?

Reply:
It is necessary to check that the tree already has an api and if not then inherit.

example native extend in es5
```js
function Class(w, h) {
    this.width = w
    this.height = h
}
Class.prototype.area = function() {
    return this.width * this.height
}

function newClass(x, y, w, h) {
    Class.call(this, w, h) // Inherit the Class property
    this.x = x
    this.y = y
}
newClass.prototype = new Class()
delete newClass.prototype.width
delete newClass.prototype.height

newClass.prototype.constructor = newClass

var obj = new newClass(1, 2, 3, 4)
```